docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,72 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+All :doc:`organization owners </commercial/subscriptions>` can manage the subscription for that organization.
+It's easy in this dashboard to achieve a number of common tasks:
+
+* Update your credit card information.
+* Upgrade, downgrade, or cancel your plan.
+* View, download, and pay invoices.
+* Add additional tax (VAT/EIN) or contact email addresses on your invoices.
+
+You can always find our most up to date pricing information on our `pricing page <https://about.readthedocs.com/pricing/>`_.
+
+Managing your subscription
+--------------------------
+
+* Go to https://readthedocs.com/organizations/.
+* Select your organization from this list.
+* Select :guilabel:`Settings` at the top of the page.
+* Select the :guilabel:`Subscription` tab on the left.
+* Click :guilabel:`Manage Subscription`.
+
+This action will take you to the Stripe billing portal where you can manage your subscription.
+
+.. note::
+    You will need to be an organization owner to view subscription information.
+    If you do not have permission,
+    you can ask one of your existing organization owners to make any required changes.
+
+Cancelling your subscription
+----------------------------
+
+Cancelling your subscription can be done in the same way as the above instructions.
+You will be billed for the remainder of the current billing period,
+and then not charged again.
+
+**We can not cancel subscriptions just by sending us an email,
+as email is an insecure method of verifying a user's identity.**
+If you email us about this,
+we require you to verify your identity by logging into your Read the Docs account and submitting an official support request there.
+
+Billing information
+-------------------
+
+We provide both monthly and annual subscriptions for all plans.
+Annual plans are given a 2 month discount compared to monthly billing.
+We only support credit card billing for our Basic & Advanced plans.
+For our Pro & Enterprise users, we support invoice-based and PO billing.
```

Comment:
I usually prefer to avoid special symbols like & in prose, `and` isn't far off and translates better.

```suggestion
We only support credit card billing for our Basic and Advanced plans.
For our Pro and Enterprise users, we support invoice-based and PO billing.
```